cmd/kots/cli/admin-console-upgrade.go

```diff
@@ -59,6 +59,13 @@ func AdminConsoleUpgradeCmd() *cobra.Command {
 			}
 
 			includeMinio := v.GetBool("with-minio")
+
+			// if the cli flag for kotsadm minio was explicitly passed, disable minio for snapshots
+			includeMinioSnapshots := v.GetBool("with-minio-snapshots")
```

Comment:
IMO, we should just use the `with-minio` flag for this as well. the customer just wants to NOT use minio, they won't care if it's just the main minio instance vs the one we deploy for snapshots.

Reply:
I think the previous thought was that it would be confusing to specify the same flag again, but I agree with you the more I think about it. cc @JustinN-Replicated

Reply:
From standup, I am OK with us consolidating under 1 flag; I will need to follow up with additional details to the email that already went out this week but it should be easy to put together and get sent out.